compiler-rt/lib/builtins/wasm/__cpp_exception.S

```diff
@@ -0,0 +1,26 @@
+//===-- __cpp_exception.S - Implement __cpp_exception.S -------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements __cpp_exception which LLVM uses to implement exception
+// handling when Wasm EH is enabled.
+//
+//===----------------------------------------------------------------------===//
+
+#ifdef __wasm_exception_handling__
```

Comment:
Weren't we going to put `__cpp_exception` into libc++abi instead of compiler-rt?

Reply:
I think compiler-rt is fine for both of these symbols, they are both very low level symbols generated at codegen time.   the most sense for both of these symbols.

Also, I'm not sure symbols like this that are arch-specific will be welcome in libc++abi (I imagine it has strict rules about naming and such, and we don't think we want rename these symbols at this point).

Reply:
I think that's fine if you don't want to wait to land this. We can always add it again later.